app/serializers/serializer_post_processing.rb

```diff
@@ -0,0 +1,18 @@
+# TODO: docs
+module SerializerPostProcessing
+  # TODO: docs
+  def serializable_object(options={})
+    super.tap {|hash|
+      modify_serializable_object(hash)
+    }
+  end
+
+  # TODO: docs
+  def modify_serializable_object(hash)
+    # TODO: having this method as an abstract overridable method is not very good,
```

Comment:
I didn't want to say that it's blocker for this PR, and I would keep 0.9 for now. This PR is a blocker for 0.7 and I'd like to have this finished first. But since you have worked with serializers it would be cool when you could try to upgrade it sometime later (probably after 0.7 is released).